Neos.Flow/Classes/Security/Context.php

```diff
@@ -618,31 +613,64 @@ public function getInterceptedRequest()
      */
     public function clearContext()
     {
+        $sessionDataContainer = $this->objectManager->get(SessionDataContainer::class);
+        $sessionDataContainer->setSecurityTokens([]);
+        $sessionDataContainer->setCsrfProtectionTokens([]);
+        $sessionDataContainer->setInterceptedRequest(null);
```

Comment:
Maybe the above three lines that leak the knowledge what "clear" means for the `SessionDataContainer` could be abstracted into a `->clear()` or `->reset()` method on the container?